### PR TITLE
[#53] Add disable tracing property in config

### PR DESCRIPTION
**Pull request by corradom**
This PR solves issue #53 

Tracing can now be disabled through the property `axon.extension.tracing.enabled`, `true` by default.

### CODE REVIEW

File: tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @author Corrado Musumeci
+ * @since 4.4
```
Comment (smcvb) [inline tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java:19]: I think this bit needs to be moved to the class level javadoc just above the `@ConfigurationProperties` annotation.

File: tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Defines the properties for the Tracing extension, when automatically configured in the Application Context.
+ */
+@ConfigurationProperties(prefix = "axon.extension.tracing")
+public class TracingExtensionProperties {
+
+    /**
+     * Enables Tracing configuration for this application.
```
Comment (smcvb) [inline tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java:33]: It is good to specify what the default is here as well.

Comment (lfgcampos) [reply tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java:33]: Are we collecting those properties somewhere?
eg: ref guide, README, etc

Comment (corradom) [reply tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java:33]: good catch : you recalled me to ask @smcvb about that: in the README I found _The reference guide chapter to cover this extension is currently work in progress._
but I found this piece in the refguide [https://docs.axoniq.io/reference-guide/extensions/tracing](url)
I will update the README and the refguide with the new configuration.

Is this ok ?

Comment (smcvb) [reply tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/TracingExtensionProperties.java:33]: Ooow gladly @corradom, please adjust those sections!

**Pull request merged**